easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-13.2.0.eb

```diff
@@ -0,0 +1,44 @@
+# contributed by Guilherme Peretti-Pezzi (CSCS)
+# updated by Alex Domingo (Vrije Universiteit Brussel)
+# updated by Åke Sandgren(Umeå University)
+# Update: Petr Král (INUITS)
+easyblock = 'ConfigureMake'
+
+name = 'NTL'
+version = '11.5.1'
+
+homepage = 'https://shoup.net/ntl/'
+
+description = """NTL is a high-performance, portable C++ library providing data structures and
+algorithms for manipulating signed, arbitrary length integers, and for vectors,
+matrices, and polynomials over the integers and over finite fields."""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'libntl'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
```

Comment:
FlexiBLAS patch still looks very much necessary.

Reply:
Sorry, I don't understand what you mean. FlexiBLAS patch for `NTL`?

Reply:
Yes. The one which is present in the previous easyconfig of this.

Reply:
I don't see such patch in any of the previous easyconfigs:
- https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-12.2.0.eb
- https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-11.3.0.eb
- https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/NTL/NTL-11.5.1-GCC-11.2.0.eb
- https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/NTL/NTL-11.3.4-GCC-8.2.0-2.31.1.eb

Reply:
Sorry, i misread eb --review-pr, this was supposed to be for FLINT

Reply:
Those modifications are now handled by `sed` in `preconfigopts` of the `FLINT` easyconfig.